microsite/data/plugins/feedback.yaml

```diff
@@ -0,0 +1,10 @@
+---
+title: Feedback
+author: Red Hat
+authorUrl: https://redhat.com
+category: Engage
+description: A plugin for collecting user feedbacks for your application. # Max 170 characters
+documentation: https://github.com/janus-idp/backstage-plugins/tree/main/plugins/feedback#readme
```

Comment:
Would be great if the screenshots were not hidden, I missed them when I was looking over the documentation. The ones you have a really good and give me a good feel for the functionality right away. This is very helpful for people who are considering using your plugin so making them visible to start would be great!

Reply:
Cool, I'll update my readme as per your suggestion